### ISSUE TITLE
Insert image filename into ALT text when uploading images

### ISSUE THREAD

**Issue by kevinansfield** [TryGhost/Ghost#8462]
When inserting image markup from drag/drop or image selection uploads the filename should be inserted into the ALT text area:

Current:
`![](/content/images/2017/05/luca-bravo-207676.jpg)`

Desired:
`![luca-bravo-207676](/content/images/2017/05/luca-bravo-207676.jpg)`
